wemake_python_styleguide/violations/consistency.py

```diff
@@ -1233,3 +1235,47 @@ def some_function():
         'Found local variable that are only used in `return` statements'
     )
     code = 331
+
+
+@final
+class NumberWithMeaninglessZeroViolation(TokenizeViolation):
+    """
+    Enforce consistent octal, hex, and binary numbers.
+
+    Reasoning:
+        We need consistency to make our code readable.
+        And equal numbers might be written in almost infinite ways.
+
+    Solution:
+        Remove meaningless zeros from the number.
```

Comment:
We need to describe our logic here. Why do we allow paired zeros. See @astynax comment in the original issue.